client/components/web-preview/index.jsx

```diff
@@ -96,7 +61,11 @@ const WebPreview = React.createClass( {
 			document.documentElement.classList.add( 'no-scroll', 'is-previewing' );
 		}
 		this.props.setPreviewShowing( this.props.showPreview );
-	},
+	}
+
+	shouldComponentUpdate( nextProps, nextState ) {
+		return shallowCompare( this, nextProps, nextState );
```

Comment:
I don't think we need this because `connect` adds a pure wrapper by default ([docs](https://github.com/reactjs/react-redux/blob/master/docs/api.md#connectmapstatetoprops-mapdispatchtoprops-mergeprops-options)). Out of curiosity, was it intentional not to use [`PureComponent` from `react-pure-render/component`](https://github.com/gaearon/react-pure-render#component) (also apparently [slated for inclusion in base React](https://facebook.github.io/react/blog/2016/07/13/mixins-considered-harmful.html))?